cpp/kernels/fmha_v2/setup.py

```diff
@@ -3414,43 +3424,7 @@ def get_lname_from_kname(kname: str) -> str:
 # The source code of paged context fmha kernels are not in this repo, but we have cubins for them.
 # Other kernels are for passing CI cases.
 def modify_cubin_header(cubin_header):
-    # for paged context fmha cases
```

Comment:
you might want to revert the changes here as those left cubins are needed right now (for some accuracy reansons). And Qidi will fix it later.